air-quality-backend/system_tests/forecast_etl_suite/cams_known_grib_test.py

```diff
@@ -0,0 +1,220 @@
+        ), "o3 aqi value does not match!"
+        assert (
+                document["so2_value"] == expected_values["so2_value"]
+        ), "o2 value does not match!"
+        assert (
+                document["so2_aqi_level"] == expected_values["so2_aqi_level"]
+        ), "so2 aqi value does not match!"
+        assert (
+                document["pm10_value"] == expected_values["pm10_value"]
+        ), "pm10 value does not match!"
+        assert (
+                document["pm10_aqi_level"] == expected_values["pm10_aqi_level"]
+        ), "pm10 aqi value does not match!"
+        assert (
+                document["pm2_5_value"] == expected_values["pm2_5_value"]
+        ), "pm2.5 value does not match!"
+        assert (
+                document["pm2_5_aqi_level"] == expected_values["pm2_5_aqi_level"]
+        ), "pm2_5 aqi value does not match!"
+
+
+# aqi values
+
+
+def test__overall_aqi_level_is_between_1_and_6():
+    dict_result = get_database_data({}, "forecast_data")
+
+    for document in dict_result:
+        overall_aqi_level = document["overall_aqi_level"]
+        assert 1 <= overall_aqi_level <= 6, f" {overall_aqi_level} is out of range"
+
+
+def test__individual_aqi_levels_are_between_1_and_6():
+    dict_result = get_database_data({}, "forecast_data")
+    pollutant_keys = [
+        "no2_aqi_level",
+        "so2_aqi_level",
+        "o3_aqi_level",
+        "pm10_aqi_level",
+        "pm2_5_aqi_level",
+    ]
+    for document in dict_result:
+        for key in pollutant_keys:
+            assert 1 <= document[key] <= 6, f"{key} {document[key]} is out of range"
+
+
+def test__overall_aqi_level_is_highest_value_of_individual_pollutant_aqi_levels():
+    dict_result = get_database_data({}, "forecast_data")
+    pollutant_keys = [
+        "no2_aqi_level",
+        "so2_aqi_level",
+        "o3_aqi_level",
+        "pm10_aqi_level",
+        "pm2_5_aqi_level",
+    ]
+
+    for document in dict_result:
+        highest_aqi = max(document[key] for key in pollutant_keys)
+        assert (
+                document["overall_aqi_level"] == highest_aqi
+        ), f"{document['overall_aqi_level']} is not equal to {highest_aqi}"
+
+
+def test_that_each_document_has_location_type_city():
+    dict_result = get_database_data({}, "forecast_data")
+
+    for document in dict_result:
+        assert (
+                document["location_type"] == "city"
+        ), f"location_type '{document['location_type']}' is not a city!"
+
+
+def test_document_count_for_single_day_is_6273():
+    dict_result = get_database_data({}, "forecast_data")
+
+    assert len(dict_result) == 6273, f"Expected 6273 documents, but got {len(dict_result)}"
+
+
+def test__created_time_is_similar_to_system_time():
+    dict_result = get_database_data({}, "forecast_data")
+    now = datetime.utcnow()
+    for document in dict_result:
+        created_time = document["created_time"]
+
+        assert now - timedelta(seconds=1000) <= created_time <= now, (
```

Comment:
As you said, this might be useful as it could be slower in a deployed env, but it might be worth running the tolerance level past the team